style.css

```diff
@@ -0,0 +1,94 @@
+body {
+font-family: Roboto;
+}
+.wrapper {
+width: 80%;
+margin: auto;
+}
+.header {
+display: flex;
+justify-content: space-between;
+}
+.header__logo {
+padding: 40px 0;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/LessonsHTML/pullRequest?prid=6226470)